### PR TITLE
#168781706 -Managers should get notifications for edited requests

### PR DESCRIPTION
**Pull request by sabin18**
#### What does this PR do?
PR to allow Managers to get notifications for edited requests
#### Description of Task to be completed?
- add notification function
#### How should this be manually tested?
- clone the repository : git clone https://github.com/andela/caret-bn-backend.git
- checkout to branch get-notifications-168781706
- run sequelize db:migrate
- run sequelize db:seed:all
- use this route to edit request `PATCH :/api/v1/requests/:id.
- use this route to get notifications `GET: /api/v1/notifications/.
#### Any background context you want to provide?
`N/A`
#### What are the relevant pivotal tracker stories?
[#168781706](https://www.pivotaltracker.com/story/show/168781706)
#### Screenshots (if appropriate)
Screen Shot 2019-11-12 at 12 14 06
Screen Shot 2019-11-12 at 12 18 36

#### Questions:
`N/A`

### CODE REVIEW

**Review by alainmateso — APPROVED** [review]
Good Job Sabin

File: src/controllers/requestController.js [hunk 1]
```diff
@@ -90,23 +91,35 @@ export default class requestController {
...
         });
       });
+
       const request = await allSearch({ id });
+      const requestData = request[0].dataValues;
+
+      const notification = await notifBuilder(
+        requestData,
+        lineManager,
+        `request ${id} has been edited . Click here to view: ${APP_URL_BACKEND}/api/v1/requests/${id}.`
```
Comment (nignanthomas) [inline src/controllers/requestController.js]: Kindly edit the message into `A request has been edited...` or `The request of id: ${id} has been edited...`

File: src/routes/api/requests.js [hunk 2]
```diff
@@ -185,5 +185,4 @@ router.put('/comments/:id', validateToken, checkId, validateComment, editComment
 router.delete('/comments/:id', validateToken, checkId, deleteComment);
 router.get('/stats/', validateToken, supplierNotAllowed, catchSearchQueries, validateRequestStas, getStats);
 router.get('/:id', (req, res) => requestController.findOne(req, res));
-
```
Comment (nignanthomas) [inline src/routes/api/requests.js]: Add a line here 😏

**Review by nignanthomas — CHANGES_REQUESTED** [review]
Great job bro @sabin18 
I left some comments on your PR.

**Review by nignanthomas — APPROVED** [review]

**Pull request merged**